nautobot/core/testing/api.py

```diff
@@ -798,10 +923,16 @@ def strip_serialized_object(this_object):
             # Verify that a PATCH with some data updates that data correctly.
             response = self.client.patch(url, update_data, format="json", **self.header)
             self.assertHttpStatus(response, status.HTTP_200_OK)
+            serialized_object = response.json()
+            strip_serialized_object(serialized_object)
```

Comment:
This logic missing before meant that the `serialized_object` we were testing below is the one from the previous subsection of this test ("Send empty PATCH request" above), which we already asserted was unchanged, rather than the response to this update request as it should be.